migrations/2_deploy.js

```diff
@@ -1,4 +1,4 @@
-var tokenData = require("../test-helpers/tokenData.json");
+var tokenData = require("../test-helpers/PolygonDeploymentData.json");
```

Comment:
We should keep using `tokenData.json` in the migration scripts. As these scripts are used in the test coverage. So we need to use the consistent data.

Reply:
sure